### ISSUE TITLE
Fix the seed data for appropriate bodies

### ISSUE THREAD

**Issue by claire-hughez** [DFE-Digital/register-early-career-teachers#862]
## Description
Golden Leaf Academy shouldn't be an academy. Appropriate bodies are all teaching school hubs.

## Acceptance criteria

- [x] Rename 'Golden Leaf Academy' to 'Golden Leaf Teaching School Hub'
- [ ] Whilst we're at it, should the seed data have 'appropriate body' after it?

**Comment by peteryates** [comment]
@claire-hughez you mentioned you were going to have a go at this. It should be pretty straightforward with find and replace, just make sure to update the associated variable and references too (`golden_leaf_academy` as well as `Golden Leaf Academy` 🙂

**Comment by claire-hughez** [comment]
> @claire-hughez, the first bit is fixed in #926. I think adding 'appropriate body' after each of them might be a bit wordy.
> 
> ![image](https://private-user-images.githubusercontent.com/128088/394350106-e8ad3b1b-65e4-4488-8021-d1e63029a789.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzM4NjQ5ODIsIm5iZiI6MTczMzg2NDY4MiwicGF0aCI6Ii8xMjgwODgvMzk0MzUwMTA2LWU4YWQzYjFiLTY1ZTQtNDQ4OC04MDIxLWQxZTYzMDI5YTc4OS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIxMFQyMTA0NDJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04YTM1YWI0MGE0ZGY3M2QzOTZmZjM4NDY0ZGYwMzQ3ZGU5ZDAzMzkwM2Y3YWU2NWQ1YTMyOGFjMTBmYmYzODE2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.cM8Vb7dY8Kru-YF8hByAQBGEc7IgAb3BLjA_MIo9maY)
> 
> I can add it if you think it's helpful though.

I don't think it is helpful, I was questioning because currently it did and thought we should change it. Sorry, wasn't clear. Thanks, I'll close this!
